articles/fleet-4.17.0.md

```diff
@@ -0,0 +1,88 @@
+# Fleet 4.17.0 | Better osquery management, user engagement, improved host vitals.
+
+![Fleet 4.17.0](../website/assets/images/articles/4.17.0-cover-1600x900@2x.jpg)
+
+Fleet 4.17.0 is up and running. Check out the full [changelog](https://github.com/fleetdm/fleet/releases/tag/fleet-v4.17.0) or continue reading to get the highlights.
+
+For update instructions, see our [upgrade guide](https://fleetdm.com/docs/deploying/upgrading-fleet) in the Fleet docs.
+
+## Highlights
+- Manage osquery across your organization. 
+- Engage with your users through Fleet Desktop.
+- Battery condition added to **Host Details** for MacOS.
+
+## Manage osquery across your organization.
+**Available in Fleet Free and Fleet Premium**
+
+![Manage osquery across your organization..](../website/assets/images/articles/fleet-4.17.0-1-1600x900@2x.jpg)
+Fleet's osquery installer is out of beta! Orbit is a lightweight wrapper for osquery that automates the process of installing, updating and configuring osquery on all of your devices. Even if you're not using Fleet!
```

Comment:
```suggestion
Fleet's osquery installer (Orbit) is out of beta! Orbit is a lightweight wrapper for osquery that automates the process of installing, updating and configuring osquery on all of your devices. Even if you're not using Fleet!
```

Maybe add Orbit in parenthesis here? This way, the user isn't confused by seeing "Orbit" in the next sentence. This makes sense if Fleet's osquery installer is equivalent to Orbit. I'm not totally sure..